elastic/kibana
# [Security Solution] Update filter

## Summary

This PR is to fix
https://github.com/elastic/siem-team/issues/740 - item 48
and https://github.com/elastic/siem-team/issues/805


It’s because previously we have a filter that filters `createBy: Elastic`, but seems that it’s not case sensitive.
That’s why when users login as `elastic` user name, the templates they create are not filtered correctly.
In this PR, I remove this clause and rely on `status:immutable or active` to distinguished elastic and custom templates


How to verify this PR:

case one - login as `elastic` user:
1. go to: x-pack/test/security_solution_cypress/runner.ts
2. in line 25 change cypress:run to cypress:open
3. then in our directory run yarn cypress:run-as-ci when the cypress is open,
4. you can access the Kibana instance in port 5620 with username elastic and password changeme
5. Download this file [timelines_export (2).txt](https://github.com/elastic/kibana/files/4984279/timelines_export.2.txt)
 and rename it from `.txt` to `.ndjson`
6. import it and see if it filters correctly

case two - login as normal user:
1. start this branch normally from your local
2. do step 5 & 6 listed above
